KeycloakServer application not working anymore

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

testsuite

### Describe the bug

When doing this:
```
cd testsuite/utils
mvn exec:java -Pkeycloak-server
```

There was the error like this:
```
Caused by: java.lang.ClassNotFoundException: io.micrometer.core.instrument.MeterRegistry
    at java.net.URLClassLoader.findClass (URLClassLoader.java:445)
    at java.lang.ClassLoader.loadClass (ClassLoader.java:592)
    at java.lang.ClassLoader.loadClass (ClassLoader.java:525)
    at java.lang.Class.getDeclaredConstructors0 (Native Method)
    at java.lang.Class.privateGetDeclaredConstructors (Class.java:3373)
    at java.lang.Class.getConstructor0 (Class.java:3578)
```

It seems similar issue like https://github.com/keycloak/keycloak/pull/36293

### Version

nightly (from 2025-01-08)

### Regression

- [X] The issue is a regression

### Expected behavior

KeycloakServer starts

### Actual behavior

KeycloakServer do not start

### How to Reproduce?

See the description

### Anything else?

_No response_

--- a/testsuite/utils/pom.xml
+++ b/testsuite/utils/pom.xml
@@ -273,6 +273,11 @@
             <artifactId>mssql-jdbc</artifactId>
             <version>${mssql-jdbc.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-micrometer</artifactId>
+        </dependency>
     </dependencies>
 
 

